packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -78,12 +78,9 @@ export async function processConfigViaLegacyPlugins (projectRoot: string, legacy
 
     const cwd = path.join(projectRoot, pluginFile)
 
-    const { CYPRESS_INTERNAL_E2E_TESTING_SELF, ...env } = process.env
```

Comment:
Only thing was wondering about - was this just unnecessary code or did this need to be removed for some reason? I was looking at where it was added in https://github.com/cypress-io/cypress/pull/20786 and couldn't tell if it was necessary

Reply:
It seemed to be a no-op. I noticed because vscode was highlighting that `CYPRESS_INTERNAL_E2E_TESTING_SELF` is unused. I really can't wait til we can turn unused code detection back on :stuck_out_tongue:

Reply:
Not unused, it was destructuring it out so that it doesn't get passed down as an env var to the child processes when we're opening sub-projects during e2e testing, but I think that is maybe no longer necessary.